src/Raven.Client/Documents/Changes/ChangeNotification.cs

```diff
@@ -174,6 +174,6 @@ public class TrafficWatchChange : DatabaseChange
         public string DatabaseName { get; set; }
         public string CustomInfo { get; set; }
         public int InnerRequestsCount { get; set; }
-        public object QueryTimings { get; set; } // TODO: fix this
+        public object QueryTimings { get; set; }
```

Comment:
I think this needs an issue, it should be hidden behind an `#if`, IIRC

Reply:
creating an issue for that